src/test/groovy/hudson/plugins/gradle/injection/MavenExamples.groovy

```diff
@@ -0,0 +1,48 @@
+package hudson.plugins.gradle.injection
+
+class MavenExamples {
+    static String simplePom() {
+        '''<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+  <groupId>hudson.plugins.gradle</groupId>
+  <artifactId>maven-build-scan</artifactId>
+  <packaging>jar</packaging>
+  <version>1.0</version>
+
+  <properties>
+    <maven.compiler.source>1.8</maven.compiler.source>
+    <maven.compiler.target>1.8</maven.compiler.target>
+  </properties>
+
+</project>'''
+    }
+
+    static String buildScanExtensions() {
+        '''<?xml version="1.0" encoding="UTF-8"?>
+<extensions>
+    <extension>
+        <groupId>com.gradle</groupId>
+        <artifactId>gradle-enterprise-maven-extension</artifactId>
+        <version>1.0.2</version>
```

Comment:
Let's maybe define version as a parameter with the default value. Then we will be able to easily use different versions when needed.

Reply:
This is only used for the `BuildScanIntegrationTest`, the Maven injection test is setting up the extension using our injection mechanism. For `BuildScanIntegrationTest` I do not think it is necessary to run it with multiple GE Extension versions.

Reply:
OK. Let's leave it as is. We definitely can do it later, when/if there is a need.